Documentation/Storage-Configuration/Advanced/ceph-osd-mgmt.md

```diff
@@ -190,3 +190,29 @@ If you don't see a new OSD automatically created, restart the operator (by delet
 
 !!! note
     The OSD might have a different ID than the previous OSD that was replaced.
+
+
+## OSD Migration
+
+Ceph does not support changing certain settings on existing OSDs. To support changing these settings on an OSD, the OSD must be destroyed and re-created with the new settings. Rook will automate this by migrating only one OSD at a time. The operator waits for the data to rebalance (pgs to become `active+clean`) before migrating the next OSD. This ensures that there is no data loss. Refer [osd-migration](https://github.com/rook/rook/blob/master/design/ceph/osd-migration.md) design doc for more information. 
+
+Following scenarios are supported for OSD migration:
+
+### OSD Encryption as Day-2 operation:
+
+Existing PVC based OSDs can be migrated to enable or disable encryption. Update the cephCluster CR `spec.storage.migration.confirmation` to `yes-really-migrate-osds` and set `StorageClassDevicesets.encrypted` setting to `true` or `false` based on whether encryption needs to be enabled or disabled. For example:
```

Comment:
No need to mention the settings in the paragraph, the yaml example is more readable below
```suggestion
Existing PVC based OSDs can be migrated to enable or disable encryption. Update the `cephCluster` CR settings such as:
```